hack/verify-ends-newline.sh

```diff
@@ -28,6 +28,7 @@ function check_ends() {
     -o -iname "*.yaml" \
     -o -iname "*.yml" \
     \) \
+    -not -path ./vendor\* \
```

Comment:
https://github.com/kubernetes-sigs/kwok/blob/df4284531ca1acfdbebe07dde574adcffbcb3772/hack/update-spelling.sh#L46-L51

Follow this and do the same for other scripts 😄